odoo-shippable/scripts/build-image.sh

```diff
@@ -51,7 +51,8 @@ DPKG_DEPENDS="postgresql-9.3 postgresql-contrib-9.3 postgresql-9.5 postgresql-co
 PIP_OPTS="--upgrade \
           --no-cache-dir"
 PIP_DEPENDS_EXTRA="line-profiler watchdog coveralls diff-highlight \
-                   pg-activity virtualenv nodeenv setuptools==33.1.1"
+                   pg-activity virtualenv nodeenv setuptools==33.1.1 \
+                   html2text==2016.9.19"
```

Comment:
@moylop260 That's how we do it [here](https://github.com/Vauxoo/docker-odoo-image/blob/master/odoo100/scripts/build-image.sh#L16), the full requirement have the dependencies from the private repositories (I forgot about this)